validation/src/main/java/io/micronaut/validation/ValidatingInterceptor.java

```diff
@@ -167,4 +179,9 @@ private Object validateReturnExecutableValidator(MethodInvocationContext<Object,
     private boolean hasValidationAnnotation(MethodInvocationContext<Object, Object> context) {
         return context.hasStereotype(Validator.ANN_VALID) || context.hasStereotype(Validator.ANN_CONSTRAINT);
     }
+
+    private Class<?>[] getValidationGroups(MethodInvocationContext<Object, Object> context) {
+      final AnnotationValue<Validated> annotation = context.getAnnotation(Validated.class);
+      return annotation == null ? new Class<?>[0] : annotation.classValues("groups");
```

Comment:
Rewrite to `context.classeValues(Validated.class, "group")`

Reply:
done